windows/test.bash

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+PROTOC_VERSION="3.19.1"
+PROTOC_GEN_GO_VERSION="v1.27.1"
+PROTOC_GEN_GO_GRPC_VERSION="30dfb4b933a50fd366d7ed36ed4f71dbba2d382e"
```

Comment:
Moving to bash seems reasonable given that this seems to running well in the Windows CI environment.
Just a small note here to update the documentation for upgrading protoc for this, but otherwise I'm on board with this approach.

Reply:
Thanks for the reminder. https://github.com/bufbuild/private-docs/pull/63 will update the doc